activesupport/lib/active_support/cache.rb

```diff
@@ -844,14 +844,18 @@ module Loader
         extend self
 
         def load(payload)
+          return nil unless payload.is_a?(String)
```

Comment:
I think this line could be surprising enough to warrant a comment.

Reply:
Made it log as well.